content/zh/learn/get-started/installation.mdx

```diff
@@ -79,6 +79,25 @@ Provider 1:
 ### Relaunch browser
 </Steps>
 
+## Run WebNN on Windows ML backend (NPU)
+
+<Steps>
+### Complete the steps for CPU and GPU above
+### Launch Chrome or Edge browser with switch `--disable_webnn_for_npu=0`
+</Steps>
+
+<div className="ort-ep">
+```shell
+"%LOCALAPPDATA%\Google\Chrome SxS\Application\chrome.exe" --disable_webnn_for_npu=0
```

Comment:
Need we explain the switch flag just like what we did for DML backend? say something like "we introduced a workaround to block some buggy NPU drivers to prevent security issues, but for testing purposes, you can bypass the workaround with this switch flag.